src/scss/tabs.scss

```diff
@@ -13,9 +13,9 @@
 	display: none;
 
 	&[data-o-tabs--js] {
+		@include oTypographyDisplay($scale: -1, $line-height: 47px);
```

Comment:
This returns `font-family`, `font-size` and `line-height`, it is okay to move it above `display` and `float`. 👌